x-pack/plugins/enterprise_search/public/applications/app_search/components/engines/engines_overview.tsx

```diff
@@ -58,7 +61,7 @@ export const EnginesOverview: React.FC = () => {
     loadMetaEngines();
   }, [metaEnginesMeta.page.current]);
 
-  return (
+  return (account.role.roleType !== 'owner' || account.kibanaIsEnabled) ? (
```

Comment:
Question, why do we have the owner logic here?

Reply:
_theoretically_ it shouldn't matter, as the owner needs to access App Search to set things up before creating any additional users and roles, but, if there are any other users, we should let them through and only allow the owner to use the gated form...